content/functions/title.md

```diff
@@ -23,3 +23,12 @@ aliases: []
 ```
 {{title "BatMan"}}` → "Batman"
 ```
+
+Can be piped to combine with others.
+
+```
+{{ range $name, $items := .Site.Taxonomies.<YOUR_TAXONOMY> }}
+  <li><a href="{{ $.Site.BaseURL }}<YOUR_TAXONOMY>/{{ $name | urlize | lower }}">{{ $name  | humanize | title }} ({{ len $items }}</a>
```

Comment:
Thanks for this submission, @ronliskey. Possible to rewrite this to use the safer/less fragile `relURL` or `absURL` functions for the anchor's URL construction rather than the global config value?